index.js

```diff
@@ -108,7 +144,8 @@ function analyze (list) {
   Object.keys(condensed).sort(function (a, b) {
     return condensed[b].length - condensed[a].length
   }).forEach(function (item) {
-    console.log('* @' + item, condensed[item].length, 'commits')
+    var collab = collaborators.indexOf(item) !== -1 ? '(is a collaborator)' : ''
```

Comment:
In a later commit/PR, we can optimize and not have this checked when `argv.collaborators` is false. Not a blocker.